src/main/java/racingcar/RacingGame.java

```diff
@@ -8,43 +13,40 @@
 import static racingcar.view.ResultView.printCars;
 
 public class RacingGame {
+    private static final int INITIAL_TRY_NO = 0;
 
-    private final String carNames;
-    private final int tryNo;
-    private int gameCount;
+    private List<String> carNames;
+    private int tryNo;
 
-    public List<Car> carList = new ArrayList<>();
+    private List<Car> cars = new ArrayList<>();
 
-    public RacingGame(String carNames, int tryNo) {
+    public RacingGame(List<String> carNames, int tryNo) {
         this.carNames = carNames;
         this.tryNo = tryNo;
         makeCar();
-        printCars(carList);
+        printCars(cars);
     }
 
     private void makeCar() {
-        String[] split = carNames.split(",");
-        carList = Car.createCar(split);
+        cars = CarFactory.createCar(carNames);
     }
 
     public void race() {
-        List<Integer> ranNumList = RandomGenerator.makeRanNumList(tryNo);
-        Race.updateCarByNum(carList, ranNumList);
-        gameCount++;
+        List<Integer> ranNumList = RandomGenerator.makeRanNumList(cars.size());
+        Race.updateCarByNum(cars, ranNumList);
```

Comment:
이미 `cars`를 상태로 가지고있고 심지어 race 메서드 안에서 지역변수로 `ranNumList`도 가지고 있는 `RacingGame` 객체가 
굳이 다른 static 함수의 인자로 이 두 파라미터를 넘겨서 계산할 필요가 있을까요? 🤔

```suggestion
        int carSize = cars.size();
        for (int i = 0; i < carSize; i++) {
            updatePosition(initCarList.get(i), randomNumList.get(i));
        }
```

또한 여기서 더 나아가서 car 객체에게 직접 전진하라는 [메시지](https://tecoble.techcourse.co.kr/post/2020-04-28-ask-instead-of-getter/)를 보내면 더 좋을 것 같아요. 😉

```suggestion
        int carSize = cars.size();
        for (int i = 0; i < carSize; i++) {
            Car car = initCarList.get(i);
            car.updatePosition(randomNumList.get(i));  // 객체의 행위를 표현하는 `moveOrNot` 같은 네이밍이 좋은 것 같아요.  
        }
        return initCarList;
```